conda/cli/conda_argparse.py

```diff
@@ -106,6 +110,27 @@ def __init__(self, *args, **kwargs):
         if self.description:
             self.description += "\n\nOptions:\n"
 
+        pm = context.get_plugin_manager()
+        self._subcommands = sorted(itertools.chain(
+            *pm.hook.conda_cli_register_subcommands()
+        ), key=operator.attrgetter('name'))
+
+        # check conflicts
+        seen = []
+        for subcommand in self._subcommands:
+            if subcommand.name in seen:
+                raise PluginError(
```

Comment:
I'm not sure if this exception text is helpful to users as written, maybe add an explanation for where the subcommand came from? Generally we should be careful about low-level information in exception messages.